### ISSUE TITLE
Generate a list of contributors for each release

### ISSUE THREAD

**Issue by sbordet** [eclipse/jetty.project#1969]
Should not be too hard to parse commit logs to generate a list of contributors for each release.
I think it's a good practice for an open source project and gives credit where is due.

**Comment by joakime** [comment]
Where would this list be presented?
The VERSION.txt?
The announcement email?
The git tag?
The github release page?

**Comment by joakime** [comment]
Got an example of how other open source projects promote the contributors?

**Comment by olamy** [comment]
Samples
http://markmail.org/message/nffmspjd4zsvcovk
And other sample http://www.apache.org/dist/commons/net/RELEASE-NOTES.txt this one use an other model: 
* issue 7777 bla bla bla. Thanks to Foo Bar

**Comment by sbordet** [comment]
Presented in the release notes.

Linux does it, Redis, etc.

**Comment by rk1165** [comment]
@sbordet , @olamy 
We can use one command in git to get the author names. It is akin to `git log --format="%aN"`. Can this be used to generate a list of contributors?

**Comment by sbordet** [comment]
@rk1165 I think @joakime has addressed this issue for the forthcoming releases.

**Comment by rk1165** [comment]
Ok @sbordet

**Comment by joakime** [comment]
See last 10.0.0 release notes for example.

https://github.com/eclipse/jetty.project/releases/tag/jetty-10.0.0

Considering this issue closed.
